lib/ansible/utils/collection_loader/_collection_finder.py

```diff
@@ -1173,10 +1173,11 @@ def _get_collection_name_from_path(path):
     path = _to_text(os.path.abspath(_to_bytes(path)))
 
     path_parts = path.split('/')
-    if path_parts.count('ansible_collections') != 1:
+    if path_parts.count('ansible_collections') < 1:
```

Comment:
Why not simply check
```suggestion
    if 'ansible_collections' not in path_parts:
```
? This would be more efficient.